docs/src/getstarted/overview.md

```diff
@@ -0,0 +1,140 @@
+
+## Transactions
+A transaction is the fundamental unit of activity on the Solana blockchain: it's a signed data structure that contains instructions for the network to perform a particular operation like transferring tokens.
+
+You need a transaction to create, update or delete data on-chain. You can read data without a transaction.
+
+All transactions interact with programs on the network - these can be system programs or user built programs. Transactions tell the program what they want to do with a bunch of instructions, and if they're valid, the program will execute them and update the state of the blockchain. Think of it like a write command that can be rejected if certain conditions aren't met.
+
+Here's a visual representation of what a transaction contains:  
+![Visual layout of a transaction](/img/transaction.svg)
+
+- Signatures: An array of digital signatures from the transaction's signers. 
+- Message: The actual instructions that the transaction is issuing to the network.
+  - Message header: 3 `uint8s` describing how many accounts will sign the payload, how many won’t, and how many are read-only.
+  - Account addresses: an array of addresses of the accounts that will be used in the transaction. 
+  - Recent blockhash: a unique value that identifies a recent block - this ensures the transaction is not too old and is not re-processed.
+  - Instructions: which program to call, which accounts to use, and any additional data needed for the program to execute the instruction.
+
+Transactions can be created and signed using clients via SDKs, or even on-chain programs.
+
+You can learn more about transactions [here](../developing/programming-model/transactions.md).
+
+### Instructions
+Instructions are the most basic operational unit on Solana. A transaction can contain one or more instructions. Instructions are executed sequentially in the order they are provided in the transaction by programs on the blockchain. If any part of an instruction fails, the entire transaction will fail. 
+
+Here's what an instruction looks like:  
+
+| Item | Description         |
+|------|---------------------------------------------------------------------------------------|
+| `Program ID`               | The ID of the program being called                              |
+| `Accounts`                 | The accounts that the instruction wants to read or modify       |
+| `Data`          | Input data provided to the program as additional information or parameters in the format of a byte array  |
+ 
+You can read more about instructions [here](../developing/programming-model/transactions#instructions).
+
+### Transaction Fees
+Every time you submit a transaction, somebody on the network is providing space and processing power to make it happen. To facilitate this, transactions on Solana require a fee to be paid in SOL. This fee is paid to the validators who process the transaction.
```

Comment:
fees are paid in units of lamports. can you make that more clear and also briefly describe the difference between lamports and the SOL token